src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -5369,7 +5416,7 @@ def set_up_sku(self, mc: ManagedCluster) -> ManagedCluster:
         """
         self._ensure_mc(mc)
 
-        if self.context.get_uptime_sla():
+        if self.context.get_uptime_sla() or self.context.get_tier() == "standard":
             mc.sku = self.models.ManagedClusterSKU(
```

Comment:
After new SDK is released and adopted, the model used here should be updated?

Reply:
Will stay the same, since we introduced the breaking change for Feb release.